### ISSUE TITLE
Command parameters are lost on update

### ISSUE THREAD

**Issue by mdolbin** [devicehive/devicehive-java-server#129]
As reported by @Nikolay-Kha - command parameters are lost when command is updated.

**Steps to reproduce:**
* Create new command with some set of parameters, status is optional
* Get command - parameters are set correctly
* Update command's status, for example
* Get command - parameters field is null

**Bash script to reproduce this issue**
```
#Defect #129 test create-update command script
API_BASE="http://127.0.0.1:8080/dh/rest"
ACC_KEY="1jwKgLYi/CdfBTI9KByfYxwyQ6HUIEfnGSgakdpFjgk="
DEVICE_ID="e50d6085-2aba-48e9-b1c3-73c673e414be"
CREATE_COMMAND='{"command":"defect #129 test command", "parameters":{"param1":"value1", "testObj":{"testArr":[]}}}'
UPDATE_COMMAND='{"status":"Done"}'

ECHO "### Creating new command ###"
ECHO "==> [$(date)] POST: $API_BASE/device/$DEVICE_ID/command/ -d $CREATE_COMMAND"
CREATE_RESPONSE=$(curl -sX POST --header "Content-Type: application/json" --header "Accept: application/json" --header "Authorization: Bearer $ACC_KEY" "$API_BASE/device/$DEVICE_ID/command/" -d "$CREATE_COMMAND")
COMMAND_ID=$(echo "$CREATE_RESPONSE" | pcregrep -o1 '"id": ([0-9]+)')
ECHO "<== [$(date)] POST: $CREATE_RESPONSE"
ECHO "Command id: $COMMAND_ID"

ECHO ""
ECHO ""

ECHO "### Getting command #$COMMAND_ID ###"
ECHO "==> [$(date)] GET: $API_BASE/device/$DEVICE_ID/command/$COMMAND_ID"
GET_RESPONSE=$(curl -sX GET --header "Accept: application/json" --header "Authorization: Bearer 1jwKgLYi/CdfBTI9KByfYxwyQ6HUIEfnGSgakdpFjgk=" "$API_BASE/device/$DEVICE_ID/command/$COMMAND_ID")
ECHO "<== [$(date)] GET: $GET_RESPONSE"

ECHO ""
ECHO ""

ECHO "### Updating command #$COMMAND_ID ###"
ECHO "==> [$(date)] PUT: $API_BASE/device/$DEVICE_ID/command/$COMMAND_ID -d $UPDATE_COMMAND"
PUT_RESPONSE=$(curl -sX PUT --header "Content-Type: application/json" --header "Accept: application/json" --header "Authorization: Bearer 1jwKgLYi/CdfBTI9KByfYxwyQ6HUIEfnGSgakdpFjgk=" "$API_BASE/device/$DEVICE_ID/command/$COMMAND_ID" -d "$UPDATE_COMMAND")
ECHO "<== [$(date)] PUT: $PUT_RESPONSE"

ECHO ""
ECHO ""

ECHO "### Getting command #$COMMAND_ID ###"
ECHO "==> [$(date)] GET: $API_BASE/device/$DEVICE_ID/command/$COMMAND_ID"
GET_RESPONSE=$(curl -sX GET --header "Accept: application/json" --header "Authorization: Bearer 1jwKgLYi/CdfBTI9KByfYxwyQ6HUIEfnGSgakdpFjgk=" "$API_BASE/device/$DEVICE_ID/command/$COMMAND_ID")
ECHO "<== [$(date)] GET: $GET_RESPONSE"
```

**Comment by mdolbin** [comment]
@Nikolay-Kha Please, let me know if you can reproduce this issue.

**Comment by devrn** [comment]
I checked it. Issue is still present on branch origin/defect/129
here is a screenshot. Update messages are without parameters though they were with.
![screenshot from 2015-10-27 00 24 02](https://cloud.githubusercontent.com/assets/11719580/10743024/979a2d88-7c41-11e5-94de-33da76f20aeb.png)

**Comment by devrn** [comment]
looks like something was wrong with my build system. Sorry. I rechecked and rebuild server and now it works.

**Comment by zubrabubra** [comment]
merge fixes, closing #129
